templates/base/config/jest.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    rootDir: '../',
+    moduleDirectories: ['node_modules'],
+    testPathIgnorePatterns: ['<rootDir>/dist', '<rootDir>/out/'],
+    clearMocks: true,
+    collectCoverage: true,
```

Comment:
lets keep coverage thresholds configured as well. Even though they will be set low

Reply:
done, added a base threshold config